LibreNMS/IPMI/IPMICommand.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+/**
+ * IPMICommand.php
+ *
+ * IPMI Command
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * @link       https://www.librenms.org
+ * @copyright  2021 Trym Lund Flogard
+ * @author     Trym Lund Flogard <trym@flogard.no>
+ */
+
+namespace LibreNMS\IPMI;
+
+use Exception;
+use LibreNMS\Proc;
```

Comment:
Please use \Symfony\Component\Process\Process, LibreNMS\Proc is deprecated (should probably mark the class)

Reply:
Fixed in 00abd18